### PR TITLE
commander calibration routines initialize everything

### PR DESCRIPTION
**Pull request by dagar**
Most of this is just being paranoid and even borderline unnecessary, but there were a couple cases of possibly uninitialized variables being used (tc_locked for example).

### CODE REVIEW

**Comment by dagar** [comment]
TODO: recalibrate everything on v5 with stackcheck

File: src/modules/commander/calibration_routines.cpp [hunk 1]
```diff
@@ -803,8 +653,8 @@ int calibrate_cancel_subscribe()
 
 	if (vehicle_command_sub >= 0) {
 		// make sure we won't read any old messages
-		struct vehicle_command_s cmd;
-		bool update;
+		vehicle_command_s cmd{};
+		bool update = false;
```
Comment (julianoes) [inline src/modules/commander/calibration_routines.cpp]:
```suggestion
		bool updated = false;
```

File: src/modules/commander/accelerometer_calibration.cpp [hunk 2]
```diff
@@ -368,9 +368,9 @@ int do_accel_calibration(orb_advert_t *mavlink_log_pub)
 
 				/* update the _SCL_ terms to include the scale factor */
 				for (unsigned axis_index = 0; axis_index < 3; axis_index++) {
-					val = 1.0f;
+					float val = 1.0f;
 					(void)sprintf(str, "TC_A%u_SCL_%u", sensor_correction.accel_mapping[uorb_index], axis_index);
```
Comment (julianoes) [inline src/modules/commander/accelerometer_calibration.cpp]: Should we use `snprintf`?

File: src/modules/commander/calibration_routines.cpp (hunk 1 above)
Comment (dagar) [reply src/modules/commander/calibration_routines.cpp]: Just noticed it's also leaking the file descriptor.

**Pull request closed without merging**